app/dts/behaviors/backlight.dtsi

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+ / {
+	behaviors {
+		/omit-if-no-ref/ bl: behavior_backlight {
+			compatible = "zmk,behavior-backlight";
+			label = "BACKLIGHT";
```

Comment:
To avoid having to just change it later, the upcoming locality work in #547 has a limit on the label length (to avoid larger GATT payloads), so can we shorten this, perhaps:

```suggestion
			label = "BCKLGHT";
```